### ISSUE TITLE
Enable Content Encryption for Single-line text fields not working

### ISSUE THREAD

**Issue by joene789** [verbb/formie#1232]
### Describe the bug

With Content Encryption switched on, values for single-line text fields are stored plain text in the database

![image](https://user-images.githubusercontent.com/1153144/211379409-7c80bc9c-5fa4-4334-be7a-428b3dbc7706.png)

![image](https://user-images.githubusercontent.com/1153144/211379934-27cf8638-4542-479e-9961-fcd005eee57f.png)

### Steps to reproduce

1. 
2. 

### Form settings

- Multi-page form: No
- Submission Method: Page Reload
- Client-side Validation: No
- Custom Form Templates: No

### Craft CMS version

4.3.3

### Plugin version

2.0.18

### Multi-site?

No

**Comment by engram-design** [comment]
Fixed for the next release. To get this early, change your `verbb/formie` requirement in `composer.json` to:

```
"require": {
  "verbb/formie": "dev-craft-4 as 2.0.20",
  "...": "..."
}
```

Then run `composer update`.

**Comment by engram-design** [comment]
Fixed in [2.0.21](https://github.com/verbb/formie/releases/tag/2.0.21)
